### ISSUE TITLE
[Bug] Warning in HA - Unable to decode json content with exception `Expecting property name enclosed in double quotes

### ISSUE THREAD

**Issue by mihsu81** [Vaskivskyi/ha-asusrouter#832]
### The problem

Warning in Home Assistant:

```
Această eroare originează într-o integrare non-standard.

Logger: asusrouter.tools.readers
Source: custom_components/asusrouter/bridge.py:514
integration: AsusRouter (documentation, issues)
First occurred: 13:12:10 (1 occurrences)
Last logged: 13:12:10

Unable to decode json content with exception `Expecting property name enclosed in double quotes: line 2 column 1 (char 2)`. Please, copy this and fill in a bug report: { , "wgs_enable":"", "wgs_lanaccess":"", "wgs_addr":"", "wgs_port":"", "wgs_dns":"", "wgs_nat6":"", "wgs_psk":"", "wgs_alive":"", "wgs_priv":"", "wgs_pub":"", "wgs1_c1_enable":"", "wgs1_c1_name":"", "wgs1_c1_addr":"", "wgs1_c1_aips":"", "wgs1_c1_caips":"", "wgs1_c2_enable":"", "wgs1_c2_name":"", "wgs1_c2_addr":"", "wgs1_c2_aips":"", "wgs1_c2_caips":"", "wgs1_c3_enable":"", "wgs1_c3_name":"", "wgs1_c3_addr":"", "wgs1_c3_aips":"", "wgs1_c3_caips":"", "wgs1_c4_enable":"", "wgs1_c4_name":"", "wgs1_c4_addr":"", "wgs1_c4_aips":"", "wgs1_c4_caips":"", "wgs1_c5_enable":"", "wgs1_c5_name":"", "wgs1_c5_addr":"", "wgs1_c5_aips":"", "wgs1_c5_caips":"", "wgs1_c6_enable":"", "wgs1_c6_name":"", "wgs1_c6_addr":"", "wgs1_c6_aips":"", "wgs1_c6_caips":"", "wgs1_c7_enable":"", "wgs1_c7_name":"", "wgs1_c7_addr":"", "wgs1_c7_aips":"", "wgs1_c7_caips":"", "wgs1_c8_enable":"", "wgs1_c8_name":"", "wgs1_c8_addr":"", "wgs1_c8_aips":"", "wgs1_c8_caips":"", "wgs1_c9_enable":"", "wgs1_c9_name":"", "wgs1_c9_addr":"", "wgs1_c9_aips":"", "wgs1_c9_caips":"", "wgs1_c10_enable":"", "wgs1_c10_name":"", "wgs1_c10_addr":"", "wgs1_c10_aips":"", "wgs1_c10_caips":"" }
```

### Your device model

Lyra Mini MAP-AC1300

### Firmware type

Stock

### Firmware version

9.0.0.4.386_41615-g7c2264e

### Integration version with the issue

0.32.0

### Method of the integration installation

HACS

### What version and type of Home Assistant installation do you use

Core 2024.5.4

### Diagnostics information

[config_entry-asusrouter-ee4da7cac5c5ea007d664b1639cb4310.json](https://github.com/Vaskivskyi/ha-asusrouter/files/15405882/config_entry-asusrouter-ee4da7cac5c5ea007d664b1639cb4310.json)

**Comment by Vaskivskyi** [comment]
Hello. The issue is fixed on the backend side. It will be available in the integration this weekend (before Monday CEST)

Thank you for your report and help in making AsusRouter better! :+1:

**Comment by Vaskivskyi** [comment]
Sorry for the delay. The issue is fixed in `0.32.1` which is already available in HACS
